lib/rage/cli.rb

```diff
@@ -112,6 +113,29 @@ def console
       IRB.start
     end
 
+    desc "middleware", "List Rack middleware stack enabled for the application"
+    def middleware
+      app = ::Rack::Builder.parse_file(options[:config] || "config.ru")
+      app = app[0] if app.is_a?(Array)
+      middlewares = []
+
+      while app
```

Comment:
You might want to use `Rage.config.middleware.middlewares` instead.

Reply:
@rsamoilov - Thank you for your comment. 
What do you think about the below approach? 
```ruby
def middleware
  environment

  Rage.config.middleware.middlewares.each do |middleware|
    say "use #{middleware.first.inspect}"
  end
end
```
This method gives me the following output in the rage test app 
```
➜  rage-test-app git:(master) ✗ rage middleware
use Rage::FiberWrapper
use Rage::Reloader
use Rage::Cors
```
Is it ok to call environment here or is there any better way to initialise the Rage::Configuration so that `Rage.config` is available in the middleware method?

Reply:
Yeah, calling `environment` should be ok

Reply:
Great. Thank you.